source/frontend/src/store/rootReducer.ts

```diff
@@ -20,6 +19,5 @@ export const reducer = {
   [jwtSlice.name]: jwtSlice.reducer,
   [filterSlice.name]: filterSlice.reducer,
   [keycloakReadySlice.name]: keycloakReadySlice.reducer,
-  [mapViewZoomSlice.name]: mapViewZoomSlice.reducer,
```

Comment:
I'm guessing leafletmouseslice is dead too.

Reply:
Not sure what that one does, so it should be removed in another task

Reply:
I will log tech debt.